src/views/contributors.ts

```diff
@@ -83,6 +83,12 @@ export class ContributorsView extends BlockView<EditableBlockProps> {
     this.createEditButton()
     this.createLegend()
     this.updateSelection()
+
+    // trigger the popUp on the first insertion of the contrbutors node
+    if (this.container?.dataset.firstInitials == 'true') {
+      this.handleEdit('')
+      this.container.dataset.firstInitials = ''
+    }
```

Comment:
I think we should open the popup on node selection (i.e. `selectNode`).

Reply:
I have replied to the previous one